packages/react-icons/build.js

```diff
@@ -59,10 +56,10 @@ function createRawStylesCopy(styleFile) {
  * @param {string} baseDir
  */
 function transpileTsc(options, baseDir) {
- const {moduleFormat,outDir} = options;
+  const { moduleFormat, outDir } = options;
   console.log(`Transpiling module format [${moduleFormat}] to -> ${outDir}/`);
-  const cmd = `npx tsc -p . --module ${moduleFormat} --outDir ${outDir}`;
-  return execSync(cmd, {stdio: 'inherit', cwd: baseDir});
+  const cmd = `npx tsc -p ./tsconfig.lib.json --module ${moduleFormat} --outDir ${outDir}`;
```

Comment:
we now use dedicated tsconfig.lib.json for production transpilation and dts emmit to provide isolated environment